src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AbstractCSharpCompletionProviderTests.cs

```diff
@@ -27,36 +28,39 @@ protected override TestWorkspace CreateWorkspace(string fileContents)
             return new CSharpCompletionService(workspace, exclusiveProviders);
         }
 
-        protected override Task BaseVerifyWorkerAsync(
+        private protected override Task BaseVerifyWorkerAsync(
             string code, int position,
             string expectedItemOrNull, string expectedDescriptionOrNull,
             SourceCodeKind sourceCodeKind, bool usePreviousCharAsTrigger, bool checkForAbsence,
-            int? glyph, int? matchPriority, bool? hasSuggestionItem, string displayTextSuffix)
+            int? glyph, int? matchPriority, bool? hasSuggestionItem, string displayTextSuffix,
```

Comment:
@genlu and I will introduce a test options object for this growing list of parameters.